blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -73,6 +70,9 @@
     private int HTTP_CONNECTION_READ_IDLE_TIME_SECONDS =
             Configuration.getInstance().getIntegerProperty(HttpConfig.HTTP_CONNECTION_READ_IDLE_TIME_SECONDS);
 
+    public static boolean EXP_TOKEN_SEARCH_IMPROVEMENTS =
```

Comment:
How about if we drop the EXP part? Because when it becomes non-experimental, are we going to change the config name? Also add comment that this relates to the feature of writing hierarchical tokens to metrics_token